app/tab-troubleshoot.R

```diff
@@ -28,22 +27,21 @@ tab_troublehsoot <- tabItem(
     align = "left",
     ## Troubleshoot a workflow via Glob
     box(
-      width = 12, solidHeader = FALSE, status = "info",
+      width = 12, solidHeader = FALSE, status = "danger",
       collapsible = TRUE, collapsed = FALSE,
       title = "Troubleshoot a Workflow",
       p("When a workflow fails but no jobs were started, or there appears to be no clear reason for a workflow to have failed, this tool can provide you the entire set of workflow metadata Cromwell has for your workflow in it's raw and unprocessed (json) form. For complex workflows, this can be rather large (and ugly!)."),
       textInput(
         inputId = "troubleWorkflowID",
-        label = "Cromwell workflow id to get metadata for:",
+        label = "PROOF workflow id to get metadata for:",
```

Comment:
Better to just say 

> Workflow id to get metadata for:

consistent with abort text and is generic for those not using proof. i'll change that